operator/src/main/java/org/keycloak/operator/v2alpha1/KeycloakController.java

```diff
@@ -21,15 +21,15 @@
 import io.fabric8.kubernetes.client.KubernetesClient;
 import io.javaoperatorsdk.operator.api.reconciler.*;
 import io.javaoperatorsdk.operator.api.reconciler.Constants;
+import io.javaoperatorsdk.operator.junit.KubernetesClientAware;
 import org.jboss.logging.Logger;
 import org.keycloak.operator.v2alpha1.crds.Keycloak;
 import org.keycloak.operator.v2alpha1.crds.KeycloakStatus;
 
 @ControllerConfiguration(namespaces = Constants.WATCH_CURRENT_NAMESPACE, finalizerName = Constants.NO_FINALIZER)
-public class KeycloakController implements Reconciler<Keycloak> {
+public class KeycloakController implements Reconciler<Keycloak>, KubernetesClientAware {
 
-    @Inject
-    Logger logger;
+    private static final Logger logger = Logger.getLogger(KeycloakController.class);
```

Comment:
Is there an issue with Quarkus CDI when running tests? My Deployment e.g. [consumes even config through CDI](https://github.com/keycloak/keycloak/blob/8b5af1a6ac7ff898e2029365e5e6a5a45da6e041/operator/src/main/java/org/keycloak/operator/v2alpha1/KeycloakController.java#L61-L62). There must be a way to test it.